frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDrawerRightTabs.tsx

```diff
@@ -22,34 +20,21 @@ enum PipelineRunNodeTabs {
 const PipelineRunNodeTabsTitles = {
   [PipelineRunNodeTabs.INPUT_OUTPUT]: 'Input / Output',
   [PipelineRunNodeTabs.DETAILS]: 'Details',
-  [PipelineRunNodeTabs.VOLUMES]: 'Volumes',
   [PipelineRunNodeTabs.LOGS]: 'Logs',
 };
 
 type PipelineRunDrawerRightTabsProps = {
-  task: PipelineRunTaskDetails;
-  taskReferences: TaskReferenceMap;
-  parameters?: PipelineRunTaskParam[];
+  task: PipelineTask;
 };
 
-const PipelineRunDrawerRightTabs: React.FC<PipelineRunDrawerRightTabsProps> = ({
-  task,
-  taskReferences,
-  parameters,
-}) => {
+const PipelineRunDrawerRightTabs: React.FC<PipelineRunDrawerRightTabsProps> = ({ task }) => {
   const [selection, setSelection] = React.useState(PipelineRunNodeTabs.INPUT_OUTPUT);
 
   const tabContents: Record<PipelineRunNodeTabs, React.ReactNode> = {
-    [PipelineRunNodeTabs.INPUT_OUTPUT]: (
-      <SelectedNodeInputOutputTab
-        taskReferences={taskReferences}
-        task={task}
-        parameters={parameters}
-      />
-    ),
+    [PipelineRunNodeTabs.INPUT_OUTPUT]: <SelectedNodeInputOutputTab task={task} />,
     // [PipelineRunNodeTabs.VISUALIZATIONS]: <>TBD 2</>,
     [PipelineRunNodeTabs.DETAILS]: <SelectedNodeDetailsTab task={task} />,
-    [PipelineRunNodeTabs.VOLUMES]: <SelectedNodeVolumeMountsTab task={task} />,
+    // [PipelineRunNodeTabs.VOLUMES]: <SelectedNodeVolumeMountsTab task={task} />,
     [PipelineRunNodeTabs.LOGS]: <LogsTab task={task} />,
```

Comment:
Probably not -- but that's not me I don't think 🤔 Shouldn't the sidebar be the Pipeline details one? Since it has no execution yet. I'll update that in my PR.